North-Seattle-College/ad440-winter2022-tuesday-repo
# options methods added to api

close #117 

description
* Add OPTIONS method to pass pre-flight cors error.
* Option method is run before post request

Time:
| Task Breakdown Description      | Approx. Time |
| ----------- | ----------- |
| added the route to API and enabled cores on api | 2 hour       |
| Tested api  | 1 hours        |


Test
1. If you haven't clone the repository 
2. After cloning in the terminal type npm install
3. After installation is done type npm start. The app should start to run
4. open developer tools and click on network
5. Type a sentence and click on submit
<img width="689" alt="Screen Shot 2022-03-14 at 10 49 17 PM" src="https://user-images.githubusercontent.com/74869949/158315469-27e122f1-7ef1-4818-8d06-ad4bcb6ded27.png">
6. Verify that the request went through
<img width="691" alt="Screen Shot 2022-03-14 at 11 04 00 PM" src="https://user-images.githubusercontent.com/74869949/158316934-a2ee7c52-6df0-4425-945b-47c5b5c1742b.png">
